src/Features/Core/Portable/PullMemberUp/IMembersPullerService.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.  
+
+#nullable disable
```

Comment:
Remove if possible.

Reply:
this is a requirement now.  no new code can be nullable disabled.